### PR TITLE
Available jobs

### PR DESCRIPTION
**Pull request by omernu**

### CODE REVIEW

File: ultimatejobweb/migrations/test_data_jobs1.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    dependencies = [
+        ('ultimatejobweb', '0001_initial'),
+    ]
+
+    def generate_data(apps, schema_editor):
+        company1 = Company(1, '"https://careers-redhat.icims.com/jobs/search?ss=1&in_iframe=1&searchKeyword=',
+                           'logo1', 'function1')
+        company2 = Company(2, 'https://www.amazon.jobs/en/', 'logo2', 'function2')
+
```
Comment (simzacks) [inline ultimatejobweb/migrations/test_data_jobs1.py]: Neither of these 2 companies are being saved - you didn't use .save()
You are also creating two companies in the for loop below.

File: ultimatejobweb/templates/available_jobs.html [hunk 2]
```diff
@@ -61,53 +63,36 @@ <h1 class="display-3">UltimateJob</h1>
...
-                <td>Kfar Saba, IL</td>
-                <td><a href="#">click here</a></td>
-
-            </tr>
+			{% for item in job %}
+			<tr> 
+				<td>{{ item.company.company_name }}</td>
+				<td>{{ item.job_title }}</td>
+				<td>{{ item.description_url }}</td>
+			</tr>
```
Comment (simzacks) [inline ultimatejobweb/templates/available_jobs.html:38]: I would combine the job title and the url so that the title is clickable.

**Review by simzacks — CHANGES_REQUESTED** [review]

File: ultimatejobweb/migrations/test_data_jobs1.py (hunk 1 above)
Comment (Yarboa) [reply ultimatejobweb/migrations/test_data_jobs1.py]: @omernu Please consider working with objects
.....
        companies = [
          (company1, 'https://careers-redhat.icims.com/jobs/search?ss=1&in_iframe=1&searchKeyword=',
           'logo1', 'function1'),
          (company2, 'https://www.amazon.jobs/en/', 'logo2', 'function2'),
        ]
......
            for company, job_title, description_url in jobs:
                company_n = Company.objects.get(company_name=company)
                Job(company=company_n, job_title=job_title, description_url=description_url).save()

File: ultimatejobweb/migrations/test_data_jobs1.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+          ('Amazon', 'https://www.amazon.jobs/en/', 'logo2', 'function2'),
+        ]
+
+        with transaction.atomic():
+            for company_name, company_search_url, company_logo, function_name in companies:
+                Company(company_name=company_name, company_search_url=company_search_url, company_logo=company_logo,
+                        function_name=function_name).save()
+
+            for company, job_title, description_url in jobs:
+                Job(company=company, job_title=job_title, description_url=description_url).save()
```
Comment (Yarboa) [inline ultimatejobweb/migrations/test_data_jobs1.py]: See Insert method above

File: ultimatejobweb/migrations/test_data_jobs1.py (hunk 1 above)
Comment (simzacks) [reply ultimatejobweb/migrations/test_data_jobs1.py]: Lets get this resolved so we can merge the PR

**Review by simzacks — APPROVED** [review]

**Comment by simzacks** [comment]
you have conflicts. rebase and submit and I'll merge

**Pull request merged**